cantabular-import/README.md

```diff
@@ -36,4 +33,16 @@ If you need to make adjustments to compose files etc, you can just
 run `docker-compose up -d` and docker-compose will automatically detect 
 which services need rebuilding (no need to bring everything down first).
 
+## Known Issues ##
+
+dp-dataset-api requires a connection to a graph db in order to not complain
+at healthcheck time. As it stands the service isn't configured to be able
+to access the ssh tunnel to Neptune from the container's host machine.
+
+This doesn't prevent the Cantabular import journey from working but it does
+produce an error log.
+
+Either dp-dataset-api needs to be able to be configured to not require a
+graph db connection or the container needs to be configured to access the
+host environments localhost.
```

Comment:
Should there be a PREREQUISITES section that lists all manual steps to set up an environment for local testing that includes doing the following in a separate terminal to forward connections to neptune:
dp ssh develop web 1 -p 8182:develop-neptune-cluster.cluster-cpviojtnaxsj.eu-west-1.neptune.amazonaws.com:8182

Reply:
Unfortunately this isn't enough, like I said the container will need to be configured to access the localhost outside the container.. I haven't nailed it down yet and I'm not sure that's even the right approach, to have to have a connection to Neptune for a journey that doesn't need it. Would prefer to add an environment variable in dp-dataset-api to skip the graph db health check but need to get other opinions really

Reply:
Therefore, in the mean time ... it must be a good idea to add the suggested instructions to avoid others getting confused with the errors ...
(others in the future who have not worked on the code, and to save them time with what will on the surface look like a problem, be missleading and wasting time)